src/extension.ts

```diff
@@ -16,6 +16,11 @@ import { multimanifestmodule } from './multimanifestmodule';
 import { authextension } from './authextension';
 import { StatusMessages } from './statusMessages';
 import { DepOutputChannel } from './DepOutputChannel';
+import { stackanalysismodule } from './stackanalysismodule';
+
+const packageJSON = stackanalysismodule.packageJSON;
+const version = stackanalysismodule.version;
+process.env['CUSTOM_HEADER'] = '{"plugin_version":"' + version + '", "security_vendor":"' + stackanalysismodule.vendor + '"}';

```

Comment:
put this into activate function, specifically after `authorize_f8_analytics` promise resolves.